tensorboard/util/grpc_util_test.py

```diff
@@ -163,89 +162,53 @@ def handler(request, context):
 
 
 class AsyncCallWithRetriesTest(tb_test.TestCase):
-    def test_aync_call_with_retries_invokes_callback(self):
+    def test_aync_call_with_retries_completes(self):
```

Comment:
This seems essentially equivalent to the `_succeeds` test now?  Maybe no need to have both.

Reply:
agreed.  removed.